notebooks/MAST/downloading_transiting_planet_data/downloading_exoplanet_data.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>This is really nicely explained. (But missing the link as intended). </p><p> </p><p>However I'm a little surprised because I thought we were jumping straight to requesting the list of observations of WASP-12 b through the MAST API. I now see we were performing a similar step to what was done in the first part of Method 1: asking MAST to resolve the name for us and make sure we have the right target. Looking ahead, it seems like our goal with this was to get the correct RA and Dec and then we will search by coordinate. So this could be explained above at the start of this section. (And maybe also split into subsections, so that will show up as such in the workflow table of contents.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>